src/sagas.js

```diff
@@ -354,7 +354,12 @@ export function* fetchSingleVm (action) {
       internalVm.sessions = yield fetchVmSessions({ vmId: internalVm.id })
       // TODO: Support <4.2 for snapshots?
     }
-
+    for (let i in internalVm.snapshots) {
```

Comment:
Can this be done with `'snapshots.disks'` or similar on the `vmFetchAdditionalList` list (which translates to `?follow=...` on the API request)?  Then add the disks transform to the `Snapshot.toInternal` transform...

As is, if a VM has a lot of snapshots, it'll make a lot of requests.

If the disks cannot be loaded by a follow, I recommend you open a BZ on the rest api to get it added!  The fewer requests the better.  Leave a TODO note in the saga explaining this.

Reply:
Unfortunately `'snapshots.disks'` doesn't work.

Reply:
Are there other `follow` options that do work?

If the snapshot's disks cannot be loaded by a follow, I recommend you open a BZ on the rest api to get it added!